File: 백준/Bronze/2566. 최댓값/최댓값.py
max_num = 0
col = 0
row = 0

for i in range(9):
    row_num = list(map(int,input().split()))
    if max(row_num) > max_num:
        max_num =max(row_num)
        row = i
        col = row_num.index(max_num)

print(max_num)
print(row+1,col+1)

File: 백준/Bronze/2566. 최댓값/최댓값.js
// 백준 : 최댓값
const input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n').map(x => x.split(' ').map(Number));
let max = 0 ;
let arr = [] ;

for(let i = 0 ; i < 9 ; i++){
    for(let j = 0 ; j < 9 ; j++){
        if(input[i][j] >= max){
            max = input[i][j];
            arr = [i+1,j+1];
        } 
    }
}
console.log(max);
console.log(arr.join(' '));